src/envs/splunk.env

```diff
@@ -0,0 +1,15 @@
+############################################################
+##
+## SPLUNK
+##
+############################################################
+# Put overrides in docker/.env, not here
+
+
+##----------------------------------------------------------
+
+
+### Splunk credentials. See splunk.py and https://github.com/splunk/splunk-sdk-python for more details
+SPLUNK_USERNAME=""
```

Comment:
@rjurney - please comment these out, and also add a comment that these are overridden if they are set in `.env` to avoid any confusion

Reply:
still pending right?

Reply:
@DataBoyTX can these be unset by default?

Reply:
(does the demo gracefully skip if so, or break gak start?)